geodiff/src/geodiff.h

```diff
@@ -103,6 +103,17 @@ GEODIFF_EXPORT int GEODIFF_CX_setLoggerCallback( GEODIFF_ContextH contextHandle,
  */
 GEODIFF_EXPORT int GEODIFF_CX_setMaximumLoggerLevel( GEODIFF_ContextH contextHandle, GEODIFF_LoggerLevel maxLogLevel );
 
+/**
+ * Set list of tables to exclude from geodiff operations. Once defined, these tables
+ * will be excluded from the following operations: create changeset, apply changeset,
+ * rebase, get database schema, dump database contents, copy database between different
+ * drivers.
+ *
+ * Tables passes as semicolon separated list, e.g. "table1;table2;...;tableN".
+ * If empty string is passed, list will be reset.
+ */
+GEODIFF_EXPORT int GEODIFF_CX_setTablesToSkip( GEODIFF_ContextH contextHandle, const char *tables );
```

Comment:
I would suggest to have the list of tables passed just like in the concat API call - first a number of tables and then an array of table names (with no parsing from a single string):
```
GEODIFF_EXPORT int GEODIFF_concatChanges(
  int inputChangesetsCount,
  const char **inputChangesets,
  const char *outputChangeset
);
```
This should be more consistent with the rest of the API and also more future proof (e.g. if ever someone in the future would have tables with semicolons in names :-) )